bedrock/base/templates/includes/banners/basic.html

```diff
@@ -5,7 +5,7 @@
 #}
 
 {# Exclude banners from search result snippets using `data-nosnippet` (issue 8739) #}
-<aside class="c-banner hide-from-legacy-ie {% block banner_class %}{% endblock %}" id="{% block banner_id %}{% endblock %}" data-nosnippet="true">
+<aside class="c-banner hide-from-legacy-ie {% block banner_class %}{% endblock %}" id="{% block banner_id %}{% endblock %}" aria-label="{{ ftl('ui-banner-label') }}" data-nosnippet="true">
```

Comment:
You may need to include something like the banner_id inside the label as well, in case more banners are shown around the page and the aside/complementary role ambiguity returns with more elements using the same accessible name.

Reply:
(In case of the other banner include I was looking into linking the id of the banner_title h2 instead, but haven't gone through the past few instances to check what kind of text appears there, if it makes any AT/descriptive sense…)

Reply:
We have a general rule of only one banner per page, so I don't think anything more complicated is needed here, but thanks for the suggestion.

Reply:
Oh right, it's not hidden just in CSS, there's an actual condition for it in the template, true! In that case could you also add it to firefox-app-store-banner?

https://github.com/mozilla/bedrock/blob/51f89f0ccf0904821c2ca28ef160981335a18dc2/bedrock/base/templates/includes/banners/mobile/firefox-app-store.html#L10-L11

(It's probably not visible to the UA as it runs from desktop with desktop UAstring even for mobile tests? – But it's the same case.)